modules/planning/conf/traffic_rule_config.pb.txt

```diff
@@ -63,6 +63,16 @@ config: {
   }
 }
 
+config: {
+  rule_id: PULL_OVER
+  enabled: true
+  pull_over {
+    max_stop_deceleration: 2.0
+    stop_distance: 0.5
```

Comment:
Does it mean the ego vehicle stops at 0.5 meters before the destination ENU point?

Reply:
yes.  it's the distance adc keeps from virtual stop wall when adc stops.